pkg/harvester/l10n/zh-hans.yaml

```diff
@@ -1278,6 +1278,7 @@ advancedSettings:
     'harv-additional-ca': '用于 TLS 验证的自定义 CA 根证书。'
     'harv-overcommit-config': '资源超额配置。'
     'harv-support-bundle-timeout': 'Support Bundle 超时配置（单位：分钟），你可以使用 0 来禁用超时。'
+    'harv-support-bundle-expiration': 'Support Bundle 過期配置（单位：分钟）。'
```

Comment:
supposed to be in simplified Chinese. @Yu-Jack we don't maintain the translation anymore, just do our best effort.